class-wxr-importer.php

```diff
@@ -1541,9 +1546,53 @@ protected function process_term( $data, $meta ) {
 			$original_id,
 			$term_id
 		) );
+		
+		$this->process_term_meta( $meta, $term_id, $term );
 
 		do_action( 'wp_import_insert_term', $term_id, $data );
 	}
+	
+	/**
+	 * Process and import term meta items.
+	 *
+	 * @param array $meta List of meta data arrays
+	 * @param int $term_id Term ID to associate with
+	 * @param array $term Term data
+	 * @return int|bool Number of meta items imported on success, false otherwise.
+	 */
+	protected function process_term_meta( $meta, $term_id, $term ) {
+		if ( empty( $meta ) ) {
+			return true;
+		}
+	
+		foreach ( $meta as $meta_item ) {
+			/**
+			 * Pre-process term meta data.
+			 *
+			 * @param array $meta_item Meta data. (Return empty to skip.)
+			 * @param int $term_id Term the meta is attached to.
+			 */
+			$meta_item = apply_filters( 'wxr_importer.pre_process.term_meta', $meta_item, $term_id );
```

Comment:
Happy too, but I wrote this to be consistent with post meta, so we ought to change https://github.com/humanmade/WordPress-Importer/blob/master/class-wxr-importer.php#L1146 too.

(Is backwards compatibility a concern here? If not, we could just replace `$term_id` with `$term`, and similarly for posts).